### PR TITLE
test: Flaky test fix

### PR DESCRIPTION
**Pull request by Vijetha-Kaja**
## Description

**Fixed below flaky tests**
- Button_onClickAction_spec.js
- DeleteWorkspace_spec.js
- LeaveWorkspaceTest_spec.js
- WorkspaceSettings_validation_spec.js
- MongoDBShoppingCart_spec.js
- GitImport_spec.js
- Basic_spec.js

## Type of change

- Flaky test fix

## How Has This Been Tested?
- Cypress test runs

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Widgets/Button/Button_onClickAction_spec.js [hunk 1]
```diff
@@ -163,6 +163,12 @@ describe("Button Widget Functionality", function() {
     // Clicking the button to verify the success message
     cy.get(publishPage.buttonWidget).click();
     cy.wait(3000);
+    cy.get("body").then(($ele) => {
+      if ($ele.find(widgetsPage.apiCallToast).length<=0) {
+        cy.get(publishPage.buttonWidget).click();
+        cy.wait(3000);
+      }
+    });
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Widgets/Button/Button_onClickAction_spec.js:171]: onClick not set properly. Lets use ts helper. propPane.UpdatePropertyFieldValue()

File: app/client/cypress/support/WorkspaceCommands.js [hunk 2]
```diff
@@ -43,9 +43,9 @@ Cypress.Commands.add("renameWorkspace", (workspaceName, newWorkspaceName) => {
 
 Cypress.Commands.add("navigateToWorkspaceSettings", (workspaceName) => {
   cy.get(homePage.workspaceList.concat(workspaceName).concat(")"))
-    .should("be.visible")
     .first()
-    .scrollIntoView();
+    .scrollIntoView()
```
Comment (Aishwarya-U-R) [inline app/client/cypress/support/WorkspaceCommands.js:47]: All workspace related failures are fixed by these change?

Comment (Vijetha-Kaja) [reply app/client/cypress/support/WorkspaceCommands.js:47]: yes

**Review by Aishwarya-U-R — APPROVED** [review]

**Pull request merged**